adr/0017-zarf-bundle.md

```diff
@@ -0,0 +1,119 @@
+# 17. Bundles
+
+Date: 2023-06-13
+
+## Status
+
+Pending
+
+## Context
+
+Orchestrating capabilities from multiple Zarf packages into meta-packages is a current weak point for Zarf. The core of Zarf was built around components as capabilities, but as Zarf packages have scaled, there has been a need to create a new boundary to manage these capabilities efficiently.
```

Comment:
Recommended change to this line "Orchestrating capabilities from multiple Zarf packages into a single artifact(package) for deployment is not currently supported in Zarf."